packages/babel-helper-builder-react-jsx/src/index.js

```diff
@@ -172,7 +180,15 @@ You can turn on the 'throwIfNamespace' flag to bypass this warning.`,
     let _props = [];
     const objs = [];
 
-    const useBuiltIns = file.opts.useBuiltIns || false;
+    const useSpread = setDefaultValue(file.opts.useSpread, false);
+    if (typeof useSpread !== "boolean") {
+      throw new Error(
+        "transform-react-jsx currently only accepts a boolean option for " +
+          "useSpread (defaults to false)",
+      );
+    }
+
+    const useBuiltIns = setDefaultValue(file.opts.useBuiltIns, false);
```

Comment:
While this makes 100% sense, it is a breaking change (for example, if you pass `0`) and we should defer it to Babel 8.

Reply:
So. Should I go back to the previous version?

Reply:
Yes 👍

Reply:
Oh ok, got it.